tests/unit/discordEphemeralResponse.test.ts

```diff
@@ -0,0 +1,27 @@
+import JSONResponse from "../../src/utils/JsonResponse";
+import { discordephemeralResponse } from "../../src/utils/discordEphemeralResponse.ts";
+
+describe("Test discordResponse function", () => {
+  const DISCORD_TOKEN = "DUMMY TOKEN";
+  it("should return a JSONResponse", () => {
+    const response = discordephemeralResponse(
+      "Dummy token message",
+      DISCORD_TOKEN
+    );
+    expect(response).toBeInstanceOf(Promise<JSONResponse>);
+  });
+  it("Should have response status 200", async () => {
+    const response = await discordephemeralResponse(
+      "Dummy token message",
+      DISCORD_TOKEN
+    );
+    expect(response?.status).toBe(200);
+  });
+  it("Should have status text defined", async () => {
+    const response = await discordephemeralResponse(
+      "Dummy token message",
+      DISCORD_TOKEN
+    );
+    expect(response?.statusText).toBeDefined;
```

Comment:
if the Act statement is same then you can place both the assert i.e., expect in one test

Reply:
I have updated, thanks Bhavika!